cli/start.go

```diff
@@ -50,14 +52,17 @@ func (s *StartCommand) addFlags() {
 
 // runStart is the entry of start command.
 func (s *StartCommand) runStart(args []string) error {
-	container := args[0]
-
-	// attach to io.
 	ctx := context.Background()
 	apiClient := s.cli.Client()
 
-	var wait chan struct{}
+	// attach to io.
 	if s.attach || s.stdin {
+		var wait chan struct{}
+		// We're going to attach to a container, we should make sure we only have one container.
```

Comment:
Change it to "If We're going to attach to a container, we should make sure we only have one container."